lms/static/js/groups/views/cohort_discussions_inline.js

```diff
@@ -17,10 +17,11 @@
                     },
 
                     render: function () {
-                        var alwaysCohortInlineDiscussions = this.cohortSettings.get('always_cohort_inline_discussions');
+                        var alwaysCohortInlineDiscussions = this.cohortSettings.get('always_cohort_inline_discussions'),
+                            inline_discussions = this.model.get('inline_discussions');
 
                         this.$('.cohort-inline-discussions-nav').html(this.template({
-                            inlineDiscussionTopics: this.getInlineDiscussionsHtml(this.model.get('inline_discussions')),
+                            inlineDiscussionTopicsHtml: this.getInlineDiscussionsHtml(inline_discussions),
```

Comment:
To clarify, I **can't** use HtmlUtils.ensureHtml unless I first wrap the HTML as an HTML snippet, correct?